src/GitVersionCore/Helpers/EnvironmentHelper.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
+using System;
 
 namespace GitVersion.Helpers
 {
     public class EnvironmentHelper
```

Comment:
Seems like we already have a sort of abstraction around `System.Environment`, we've just not completed it. I'm also not a fan of the word `Helper`. What about just `Env`?

Reply:
It was not possible to inject the IEnvironment into an extension method where is is used - StringFormatWithExtension.FormatWith<T>(this string template, T source), I removed the `Helper` class as it was the only usage